api/rpc/logs/logs.go

```diff
@@ -154,8 +153,8 @@ func (s *Server) Get(ctx context.Context, in *GetRequest) (*GetReply, error) {
 func (s *Server) GetStream(in *GetRequest, stream Logs_GetStreamServer) error {
 	log.Println("rpc-logs: GetStream", in.String())
 
-	sub, err := s.NatsStreaming.GetClient().Subscribe(amp.NatsLogsTopic, func(msg *stan.Msg) {
-		entry, err := parseProtoLogEntry(msg.Data)
+	sub, err := s.MessageQueuer.Subscribe(amp.MQLogsTopic, func(msg []byte) {
```

Comment:
MQLogsTopic ... I don't think the `MQ` part is needed, and `Topic` is part of the Kafka legacy. I think it would be better to rename to `LogsQueue` (at least once you at message queue support in addition to pub/sub, as mentioned in another comment in this review).